src/email-builder/template-editor/editor-canvas/classes/SocialMailSection.ts

```diff
@@ -0,0 +1,65 @@
+import { BaseMailSection } from './BaseMailSection';
+import { TemplateEditorToolCategory, MailSectionConfigurationBlock } from 'email-builder/template-editor/template-editor.types';
+import { OrientationConfigurationBlock } from 'email-builder/template-editor/right-toolbar/classes/OrientationConfigurationBlock';
+import { SocialIdConfigurationBlock } from 'email-builder/template-editor/right-toolbar/classes/SocialIdConfigurationBlock';
+
+export class SocialMailSection extends BaseMailSection {
+
+    public facebook: string;
+    public twitter: string;
+    public googlePlus: string;
+    public skype: string;
+    protected orientationConfiguration: OrientationConfigurationBlock;
+    protected socialIdConfiguration: SocialIdConfigurationBlock[];
+
+    constructor () {
+        super();
+        this.category = TemplateEditorToolCategory.Social;
+        this.orientationConfiguration = new OrientationConfigurationBlock();
+        this.socialIdConfiguration = [new SocialIdConfigurationBlock('Facebook', this.setFacebookId.bind(this)), 
```

Comment:
The app overall is nice and intuitive.
Only this social configuration part, I was not sure what to put in the UI. URL or accound id or something else?